docs/README.md

```diff
@@ -0,0 +1,80 @@
+            ~ name      = "_sip._udp.test-example.srv.test**.**.com" -> "test-example.srv"
+            # (13 unchanged attributes hidden)
+            }
+
+* If you add a CAA record, a `flags` parameter is returned in the data object. The work is being tracked in `IBM-Cloud/terraform-provider-ibm` [issue 4792](https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4792).
+
+* The returned `flags` parameter means that when you run a `terraform plan` command after a successful `terraform apply`, the output shows that the DNS record requires an update, as shown in the following example. You can ignore that message. Your infrastructure will not be affected.
+
+        # module.cis_dns_records.ibm_cis_dns_record.dns_records["test-exmple.caa/CAA"] will be updated in-place
+        ~ resource "ibm_cis_dns_record" "dns_records" {
+            ~ data        = {
+                - "flags" = "0" -> null
+                    # (2 unchanged elements hidden)
+                }
+                id        = "fcef7410xxxxxxbad23c5fd0e7581b7c:7e66xxxxxecc7e12ac908ca75445ad21:crn:v1:bluemix:public:internet-svcs:global:a/abac0df06b644axxxxx6e44f55b3880e:06240432-xxxx-40e7-9f9c-594dfbdfe208::"
+                # (12 unchanged attributes hidden)
+            }
+
+## Web Application Firewall(WAF)
+
+A Web Application Firewall (WAF) safeguards web applications by scrutinizing and overseeing HTTP traffic that flows between a web application and the internet. WAF is implemented through two rule sets: [OWASP](https://cloud.ibm.com/docs/cis?topic=cis-waf-settings#owasp-rule-set-for-waf) and [CIS](https://cloud.ibm.com/docs/cis?topic=cis-waf-settings#cis-ruleset-for-waf). For more information see, [Web Application Firewall concepts](https://cloud.ibm.com/docs/cis?topic=cis-waf-q-and-a).
+
+Currently, the [waf module](https://github.com/terraform-ibm-modules/terraform-ibm-cis/blob/main/modules/waf/) provides terraform resources to enable and disable WAF with default policies in a CIS instance.
+
+##  Distributed Denial of Service(DDOS)
+
+A distributed denial of service (DDoS) attack is a malicious attempt to disrupt normal traffic of a server, service, or network by overwhelming the target or its surrounding infrastructure with a flood of internet traffic. IBM Cloud Internet Service(CIS) provides unlimited DDoS mitigation without any additional cost. To activate the DDoS protection the following conditions must be met-
+
+  * The domain must be active.
+  * The global load balancer(GLB) or DNS records needs to be proxied as mentioned in the [documentation](https://cloud.ibm.com/docs/cis?topic=cis-about-ibm-cloud-internet-services-cis).
+  * In case of DNS records, CIS module only proxies traffic for `A`, `AAAA`, and `CNAME` records as described [here](https://cloud.ibm.com/docs/cli?topic=cli-cis-cli#dns-record).
```

Comment:
the last point requires rewording...as it is not indicating that how it is related to DDoS?